### ISSUE TITLE
Message "the comment couldn't be saved" appears in dashboard. 

### ISSUE THREAD

**Issue by ValeriaCobianISW** [publiclab/plots2#5656]
**You can reproduce this bug with the next steps:** 
-Go to notes and try to publish a 'blank' comment. 
-Go to dashboard and the error message will appear in the top of the page. 

![PRNew](https://user-images.githubusercontent.com/26575531/57152516-96883700-6d88-11e9-87a7-a3ab1882992e.gif)

**Improvements:** 
-Section comments: Evaluate when is something in the text area if anything is in the text area the button 'publish' wouldn't be enabled.

I'm going to work on this issue, for now, if you have ideas you can comment!

**Comment by ValeriaCobianISW** [comment]
@gauravano @jywarren   What do you think about evaluating in the comment section about the content in the text area,? For example, the 'publish' button will be enabled only when the text area has text.

**Comment by gauravano** [comment]
yes, disabling the Publish button in case of empty text area sounds good. Thanks!

**Comment by ValeriaCobianISW** [comment]
Hi @gauravano this issue is done, asking for review!!

**Comment by ValeriaCobianISW** [comment]
Hi @gauravano , can you review the pull request?

**Comment by amisha1072006** [comment]
Thanks for the work on this PR!
If any help is needed with testing, documentation, or follow-up changes,
I’d be happy to contribute.
